include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_utils.h

```diff
@@ -150,9 +150,9 @@ using access_mode = sycl::access_mode;
 
 // substitution for C++17 convenience types
 template <typename _T>
-using __decay_t = typename ::std::decay<_T>::type;
+using __decay_t = ::std::decay_t<_T>;
```

Comment:
In principle, I don't mind about having such a change but it makes ``__decay_t`` and ``__enable_if_t`` absolutely meaningless.

And, yeah, I don't oppose assuming if one of the following PRs would remove ``__decay_t`` and ``__enable_if_t`` completely.

Reply:
This fix already done at https://github.com/oneapi-src/oneDPL/pull/1174